<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeActorGraphicDialog</class>
 <widget class="QWidget" name="ChangeActorGraphicDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Actor Graphic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLineEdit" name="line_character">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QToolButton" name="button_battler">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="button_character">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="combo_actor"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="line_battler">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Character Graphic:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Actor:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Battler Graphic:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
